src/de/mod.rs

```diff
@@ -8,6 +8,9 @@ use serde_crate::de::IntoDeserializer;
 use serde_crate::de::Error as DeError;
 use ::SizeLimit;
 use super::{Result, Error, ErrorKind};
+use self::read::BincodeRead;
+
+pub(crate) mod read;
```

Comment:
`pub(crate)` doesn't work on stable yet so I would strongly discourage you from releasing with it.